src/lib/helpers/ManifestHelper.ts

```
@@ -0,0 +1,96 @@
+  categories: string[];
+  lang: string;
+  dir: 'ltr' | 'rtl';
+  icons: Array<{
+    src: string;
+    sizes: string;
+    type: string;
+    purpose: string;
+  }>;
+}
+
+export function generateDynamicManifest(): PWAManifest {
```

ESLint

[error] Delete `␍`

(prettier/prettier)